src/main/resources/application.yml

```diff
@@ -6,4 +6,14 @@ api:
     url: ${API_ERO_MANAGEMENT_URL}
 
 dynamodb:
-  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
\ No newline at end of file
+  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
+  scheduler-lock-table-name: ${DYNAMODB_SCHEDULER_LOCKS_TABLE_NAME}
+
+jobs:
+  enabled: true
+  print-requests:
+    name: "BatchPrintRequests"
+    cron: "${JOB_PRINT_REQUESTS_CRON:0 0 */1 * * *}" # every hour by default
```

Comment:
Looking good now. The only thing I would do (but you can do it in your next PR if you like) is to rid of this env var `JOB_PRINT_REQUESTS_CRON` altogether, both from here, and also in the readme.

Generally speaking we document env vars in the readme that are must haves - the app wont start/operate correctly without them.

In this case its an override, and we can do it without specifying the env var:
```
    cron: "0 0 */1 * * *" # every hour by default
```
If we wanted to override it we have 2 options - we can either commit an application yml for the environment with just the override. EG:
```application-prod.yml
jobs:
  print-requests:
    cron: "0 0 */2 * * *}"
```
or we can set an environment variable that corresponds to the property path:
```
JOBS_PRINT_REQUESTS_CRON="0 0 */3 * * *"
```
Spring will do magic and map the env var to the correct property. Magic 😁 

So, as mentioned, generally speaking we only use env vars when we need to set the value per environment, and it is only those env vars that we document in the readme (the essential ones)